### ISSUE TITLE
Background turning black when opening new tab

### ISSUE THREAD

**Issue by christianheinrichs** [Guake/guake#932]
Hello,

recently I installed the _guake_ terminal on my Debian machine and it works fine as expected. Used this program for some time now.

However, on my current OS (Debian Jessie 64-bit, running GNOME), whenever opening a new tab with <kbd>CTRL</kbd>+<kbd>Shift</kbd>+<kbd>T</kbd>, the background turns black instead of the normal transparent, so I have to manually close and open _guake_ again by typing in `ps aux | grep guake`, find the PID, close it with `kill PID`, press <kbd>ALT</kbd>+<kbd>F2</kbd> and then type in `guake`. This restores the normal behaviour of _guake_.

What might be causing this?

**Comment by christianheinrichs** [comment]
About 1 day after posting this issue, I tried to resolve it myself by reinstalling guake via `apt-get`. However, for some reason that didn't work if I remember correctly. I then downloaded the source code and compiled it with the necessary build packages and dependencies and ran into the same issue.

After that, I played around with the window height and transparency setting sliders and for some reason that seems to have fixed it.

Since I am not sure whether or not this might be a _Debian Jessie_ or even _GNOME_ related bug, I am closing this issue unless there is a good reason to re-open it.
